navigation/MyStack.tsx

```diff
@@ -1,30 +1,34 @@
-import React from 'react';
-import { createStackNavigator } from '@react-navigation/stack';
-import { Home } from '../screens/Home/Home';
-import { SignUp } from '../screens/SignUp/SignUp';
-import Header from '../shared/Header';
-import SignUpHeader from '../shared/SignUpHeader';
+import React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { HomeCard } from "../screens/Home/HomeCard";
+import { SignUp } from "../screens/SignUp/SignUp";
+import Header from "../shared/Header";
+import SignUpHeader from "../shared/SignUpHeader";
 
 const Stack = createStackNavigator();
 
 const MyStack = () => {
   return (
-    <Stack.Navigator initialRouteName='Home'>
+    <Stack.Navigator initialRouteName="Home">
```

Comment:
are we using single or double quotes?

Reference:
https://www.freecodecamp.org/news/what-is-linting-and-how-can-it-save-you-time/
https://dev-yakuza.posstree.com/en/react-native/eslint-prettier-husky-lint-staged/